librz/bin/format/luac/v54/parse_54.c

```diff
@@ -466,10 +492,26 @@ RzBinInfo *lua_parse_header_54(RzBinFile *bf, st32 major, st32 minor) {
 	buffer = bf->buf;
 
 	/* read header members from work buffer */
-	ut8 luac_format = rz_buf_read8_at(buffer, LUAC_54_FORMAT_OFFSET);
-	ut8 instruction_size = rz_buf_read8_at(buffer, LUAC_54_INSTRUCTION_SIZE_OFFSET);
-	ut8 integer_size = rz_buf_read8_at(buffer, LUAC_54_INTEGER_SIZE_OFFSET);
-	ut8 number_size = rz_buf_read8_at(buffer, LUAC_54_NUMBER_SIZE_OFFSET);
+	ut8 luac_format;
```

Comment:
And here

Reply:
I am against the use of macro to hide some code pattern. The code isn't optimal, but it is easy to understand what is happening.